src/UIImage/UIImage+ColorMaskedImage.h

```diff
@@ -0,0 +1,22 @@
+//
+//  UIImage+ColorMaskedImage.h
+//  Chronability
+//
+//  Created by Logan Wright on 3/18/15.
+//  Copyright (c) 2015 IntrepidPursuits. All rights reserved.
+//
```

Comment:
My vote is to remove them now that I think about it.  I'd rather get these pulls into the project, then remove them in a new commit so that I don't have to go through each one.